_source/_posts/2024-02-20-net-scim.md

```diff
@@ -0,0 +1,1909 @@
+
+You can now go back to your SCIM Application. Navigate to **Applications** > **Applications** in the sidebar. Find your SCIM app and press on the list item to edit it. In the **Assignments** tab, click the **Assign** button and select **Assign to People** in the dropdown menu. Click the **Assign** button next to "Tom Anderson" in the user list, and press **Save and Go Back** to accept the default values.  Press the **Done** button to save your changes. On the Swagger UI, double-check that Tom is in the downstream server by making a GET /scim/v2/users request to list all current users. 
+
+### Deprovision a user
+
+Let's say Tom's contract with the company ends, and you must deprovision him from the application. 
+
+In the **Assignments** tab of the Okta SCIM application, find "Tom Anderson" and press the **X** button next to Tom's entry to unassign them from the app. This unassignment triggers Okta to send a PATCH request to your .NET SCIM server, setting the unassigned user's `active` attribute to `false` which indicates that a user's account is deactivated.
+
+We can confirm that Tom's `active` attribute is now `false` in the Swagger UI by making a request to the GET /scim/v2/users endpoint filtering for Tom's email "tom.anderson@fake.domain". 
+
+### Reprovision a user
+
+Let's say Tom gets hired full-time with the company and needs access to the application again. 
+
+To reactivate Tom's account, we will repeat the steps for assigning his Okta account to the application.  Reactivating Tom causes Okta to send a PATCH request to your scim server, setting their `active` attribute to `true`. 
+
+Again, we can confirm that Tom's account is now `active` through the Swagger UI by making a request to the GET Users endpoint filtering for Tom's email "tom.anderson@fake.domain". 
+
+### Change a user's info in Okta
+
+Let's look at one more scenario. Let's say not only has Tom returned, but he also changed his name! Worry not, as our SCIM connection will automatically propagate this profile update. Let's change Tom's name to Leo and watch how Okta sends the new name to the .NET SCIM server. 
+
+In the Okta Admin Console sidebar, navigate to **Directory** > **People**, and click on Tom's name in the user list to edit their settings. In the **Profile** tab, click **Edit** in the **Attributes** pane. Change the `firstName` field to "Leo", and update their `login` and `email` to `leo.anderson@fake.domain`. Use the **Save** button at the bottom of the page to save your changes. Again, on the Swagger UI, double-check Tom's updated name by making a GET /scim/v2/users request to list all current users.  
+
+## Troubleshooting tip:
```

Comment:
Remove colon from section headers. but I don't think this should be a section header. Consider a bold or inside a quote block